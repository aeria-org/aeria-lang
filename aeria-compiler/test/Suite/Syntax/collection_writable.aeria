collection CollectionWritableTest {
  writable {
    prop1
    prop2
    prop3
  }
}
