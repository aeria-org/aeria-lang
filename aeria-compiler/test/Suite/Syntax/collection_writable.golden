{
  "kind": "Program",
  "collections": [
    {
      "name": {
        "value": "collectionwritabletest",
        "kind": "CollectionName"
      },
      "kind": "Collection",
      "writable": [
        {
          "propertyName": {
            "value": "prop1",
            "kind": "PropertyName"
          },
          "kind": "WritableItem"
        },
        {
          "propertyName": {
            "value": "prop2",
            "kind": "PropertyName"
          },
          "kind": "WritableItem"
        },
        {
          "propertyName": {
            "value": "prop3",
            "kind": "PropertyName"
          },
          "kind": "WritableItem"
        }
      ]
    }
  ]
}
